docs/Fields.md

```diff
@@ -545,11 +580,11 @@ import { SelectField } from 'react-admin';
 
 ### Properties
 
-| Prop              | Required | Type                          | Default | Description                                                                                                                                  |
-| ----------------- | -------- | ----------------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
-| `optionText`      | Optional | `string | function | Element` | 'name'  | Name of the field to use to display the matching choice, or function returning that field name, or a React element to render for that choice |
-| `optionValue`     | Optional | `string`                      | 'id'    | Name of the field to compare to the value to find the matching choice                                                                        |
-| `translateChoice` | Optional | `Boolean`                     | true    | Whether or not the choice text should be translated                                                                                          |
+| Prop              | Required | Type                                         | Default | Description                                                                                                                                  |
+| ----------------- | -------- | -------------------------------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| `optionText`      | Optional | `string ` &#124; `Function` &#124; `Element` | 'name'  | Name of the field to use to display the matching choice, or function returning that field name, or a React element to render for that choice |
```

Comment:
why did you change that? It renders fine in the doc site

Reply:
I belive this kind of formatting is necesary (correctme if I'm wrong), That's my understanding of this commit b625602e21d98aa9af3db8194816dafecb4743e6 made by @djhi regarding  `jekyll` warnings being thrown